src/main/scala/com/fortysevendeg/scala/android/ui/pathmorphing/Layout.scala

```diff
@@ -142,30 +142,32 @@ trait Layout
                 ) <~ tableLayoutRowStyle
               ) <~ tableLayoutStyle <~ wire(colorSelectorGroup),
               w[TextView] <~ wire(strokeTitle) <~ tvText(R.string.title_select_stroke) <~ titleStyle,
-              w[SeekBar] <~ wire(strokeSelector) <~ strokeStyle <~ sbOnSeekBarChangeListener(
-                OnSeekBarChangeListenerHandler(
-                  onProgressChangedHandler = (view: SeekBar, progress: Int, fromUser: Boolean) => {
+              w[SeekBar] <~ wire(strokeSelector) <~ strokeStyle <~ MultFuncOn.SeekBarChange[SeekBar](
+                (onProgressChanged: OverrideMethodType) => {
+                  (view: SeekBar, progress: Int, fromUser: Boolean) => {
                     val stroke = progress + 1
 
-                    (strokeTitle <~ tvText(context.get.getResources().getString(R.string.title_select_stroke, stroke.toString))) ~
+                    (strokeTitle <~ tvText(context.get.getResources.getString(R.string.title_select_stroke, stroke.toString))) ~
                         (icon <~ pmdStroke(stroke dp))
                   }
-                )
+                },
+                (onStartTrackingTouch: OverrideMethodType) => (seekBar: SeekBar) => Ui(Some(seekBar)),
+                (onStopTrackingTouch: OverrideMethodType) => (seekBar: SeekBar) => Ui(Some(seekBar))
               ),
               w[TextView] <~ wire(sizeTitle) <~ tvText(R.string.title_select_size) <~ titleStyle,
-              w[SeekBar] <~ wire(sizeSelector) <~ sizeStyle <~ sbOnSeekBarChangeListener(
-                OnSeekBarChangeListenerHandler(
-                  onStopTrackingTouchHandler = (view: SeekBar) => {
-                    val Dim(width, height) = sizeOptionList(view.getProgress())
+              w[SeekBar] <~ wire(sizeSelector) <~ sizeStyle <~ MultFuncOn.SeekBarChange[SeekBar](
+                (onProgressChanged: OverrideMethodType) => (seekBar: SeekBar, progress: Int, fromUser: Boolean) => Ui(Some(seekBar)),
+                (onStartTrackingTouch: OverrideMethodType) => (seekBar: SeekBar) => Ui(Some(seekBar)),
+                (onStopTrackingTouch: OverrideMethodType) => (seekBar: SeekBar) => {
+                  val Dim(width, height) = sizeOptionList(seekBar.getProgress)
 
-                    (sizeTitle <~ tvText(context.get.getResources().getString(R.string.title_select_size, width.toString, height.toString))) ~
-                        (colorSelectorGroup <~ setNoIconImageViewWidgets) ~
-                        (sampleColor1Icon <~ pmdAnimIcon(CHECK)) ~
-                        (strokeSelector <~ sbProgress(2)) ~
-                        (icon <~ drawableStyle(width dp, height dp, 3 dp) <~ pmdColorResource(R.color.path_morph_sample_1)) ~
-                        (iconSelectorGroup <~ deactivateImageViewWidgets)
-                  }
-                )
+                  (sizeTitle <~ tvText(context.get.getResources.getString(R.string.title_select_size, width.toString, height.toString))) ~
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/47Degrees/scala-android/file/661375957/issues/source?bid=2339068&fileBranchId=2339111#l164)